docs/DeveloperGuide.adoc

```diff
@@ -295,61 +295,147 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
 [width="59%",cols="22%,<23%,<25%,<30%",options="header",]
 |=======================================================================
 |Priority |As a ... |I want to ... |So that I can...
-|`* * *` |new user |see usage instructions |refer to instructions when I forget how to use the App
+|`* * *` |user |trace all my internship application's contact | easily follow up on the application
 
-|`* * *` |user |add a new person |
+|`* * *` |user |tag each application with a status | track my internship application phase
 
-|`* * *` |user |delete a person |remove entries that I no longer need
+|`* * *` |self-reflecting user |mark what positions of internship I have been applying to | see which positions I have the best chance of getting and easily look up past internship application when applying to similar positions
 
-|`* * *` |user |find a person by name |locate details of persons without having to go through the entire list
+|`* * *` |user |set reminders for internship deadlines/appointments| make sure I do not miss any internship opportunities
 
-|`* *` |user |hide <<private-contact-detail,private contact details>> by default |minimize chance of someone else seeing them by accident
+|`* *` |user |be able to add companies I wish to apply to in a wish-list |apply to them when the window opens
 
-|`*` |user with many persons in the address book |sort persons by name |locate a person easily
+|`* *` |self-reflecting user |see at which stage my internship application failed |get a better idea of what to improve on
+
+|`* *` |future job seeker|use this program to easily reference successful applications |apply them to future endeavours
+
+|`* *` |disorganised user |store my cover letters |easily refer to them when applying for internships
+
+|`* *` |user |give a rating to each internship based on my preference |easily decide which internship to prioritise
+
+|`*` |frequent interviewee |maintain a checklist of questions to ask the interviewer |
+
+|`*` |first-time internship seeker |use the program as a guide to internship applications |learn how to start applying for an internship
```

Comment:
Could this have been more specific to describe how this program can act as a guide?